src/bms.lisp

```diff
@@ -0,0 +1,93 @@
+(defun bms-loop () {
+    (var vmin-limit)
+    (var vmax-limit)
+    (var tmax-limit)
+    (var tmin-limit -10)
+    (var cell-balance 0.5)
+    (var mosfet-tmax-limit)
+    (var config-update-time)
+    (var fault 0u32)
+    (var v-cell-support (eq (first (trap (get-bms-val 'bms-v-cell-min))) 'exit-ok))
+    (var bms-data-version-support (and (eq (first (trap (get-bms-val 'bms-data-version))) 'exit-ok) (> (get-bms-val 'bms-data-version) 0)))
```

Comment:
What does `bms-data-version-support` represent? Seems improperly named to me... Might need an explanation comment.